packages/axios-extension/src/abap/types/adt-types.ts

```diff
@@ -57,3 +57,15 @@ export interface AdtCategory {
     term: AdtCategoryTerm;
     scheme: string;
 }
+
+export interface TransportRequest {
+    transportNumber: string;
+    user: string;
+    date: string;
+    time: string;
+    description: string;
+    client: string;
+    targetSystem: string;
+    transportRequestFunction: string;
```

Comment:
Transport request function? Add JS Doc for this please? What kind of values are expected?

Reply:
Looks like a single letter from backend response. I wish I know.

Reply:
Removed properties that we are uncertain about their data format.